src/staticdata_utils.c

```diff
@@ -599,20 +599,20 @@ JL_DLLEXPORT uint8_t jl_cache_flags(void)
 {
     // OOICCDDP
     uint8_t flags = 0;
-    flags |= (jl_options.use_pkgimages & 1); // 0-bit
+    // don't use use_pkgimages here because no outputo overrides it
+    flags |= ((jl_options.outputo == NULL || jl_options.outputo[0] == '\0') | 1); // 0-bit
```

Comment:
This seems a bit odd... Are you missing the negation? And `| 1` won't it put the pkgimage bit high always?